src/main/java/lt/vu/persistence/orm/entities/Attribute.java

```diff
@@ -0,0 +1,43 @@
+package lt.vu.persistence.orm.entities;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Entity
+@Table(name = "ATTRIBUTE")
+@NamedQueries({
+    @NamedQuery(name = "Attribute.findAll", query = "select a from Attribute a"),
+    @NamedQuery(name = "Attribute.findAllByIds", query = "select a from Attribute a where a.id in (:ids)"),
+})
+@Getter @Setter
+public class Attribute implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name = "LABEL", nullable = false)
+    private String label;
+
+    @Column(name = "TYPE", nullable = false)
+    private String type;
```

Comment:
I'm wondering, why we are not using enum here? 🤔 Is it because this way is easier for frontend?

Reply:
Now that you say it, I wonder myself why this is a string 😄 This fiels is not even meant to be used in frontend. Will fix it 🙂